index.html

```diff
@@ -1162,6 +1162,11 @@ <h2>Representations</h2>
 are conformant with the fragment processing rules defined in section
 <a href="#fragment"></a> of this specification.
         </li>
+        <li>
+The representation MAY define representation-specific syntax that can
```

Comment:
do we need a definition for "representation-specific syntax" ?

Reply:
This may get editorially adjusted to "representation properties" (or whatever language is settled on there) based on PR #455.

Reply:
is CDDL "representation-specific syntax"?

Reply:
@jonnycrunch based on the relationship between CDDL and mime types, I would say that CDDL is an expression of representation specific syntax, that is more compatible with a comprehensible ADM.